docs/release-notes.rst

```diff
@@ -36,6 +36,9 @@ Changes and New Features
 
    * - Version
      - Description
+   * - 24.10.0
+     - | - Added support for NVIDIA NIC Configuration Operator deployment.
+       | - Added support for Support Single-Node OpenShift (SNO).
```

Comment:
How do we communicate the change regarding the NCP not being created by helm anymore?

Reply:
done